league/dashboard/models.py

```diff
@@ -3,35 +3,97 @@
 
 from enum import Enum
 
-from league.database import Column, Model, SurrogatePK, db, reference_col
+from league.database import (Column, Model, SurrogatePK, association_proxy, db,
+                             reference_col, relationship)
 
 Color = Enum('Color', 'white black')
 
 
+class Player(SurrogatePK, Model):
```

Comment:
Because the old way (which I agree was much more logical) made SQLAlchemy vomit. I'm pretty new to entity-mapped ORMs (and ORMs in general) so it's possible I was doing something simple wrong.